wake_t/beamline_elements.py

```diff
@@ -585,9 +585,9 @@ class PlasmaRamp():
 
     """Defines a plasma ramp."""
 
-    def __init__(self, length, plasma_dens_top, plasma_dens_down=None,
+    def __init__(self, length, plasma_dens_top=None, plasma_dens_down=None,
                  position_down=None, ramp_type='upramp',
-                 profile='inverse square', wakefield_model='blowout',
+                 profile='inverse square', wakefield_model='blowout', dens_func=None,
```

Comment:
I wouldn't define here an additional `dens_func` parameter. I would instead reuse `profile` and allow it to be either a string (as it is now) or a function of `z`.

Reply:
Good idea!

Reply:
Looks good now, thanks!